src/comet_llm/config.py

```diff
@@ -56,6 +59,10 @@ def api_key() -> Optional[str]:
     return api_key  # type: ignore
 
 
+def logging_level() -> Optional[str]:
+    return _COMET_ML_CONFIG["comet_llm.logging_level"]
```

Comment:
Looks like you need to add `# type: ignore` to avoid lint error

Reply:
It's not an Optional[str], its my bad